spec/support/authenication_helper.rb

```diff
@@ -0,0 +1,12 @@
+module AuthenticationHelper
+  def login_user
+    user = FactoryGirl.create(:user)
+    visit root_path
+
+    click_link 'Sign In'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+    user
```

Comment:
A more typical pattern for this is to have a `def login_as(user)` so that you can pass in a user with specific attributes (for example an admin user, a user with a specific username, etc). If you can justify why you'd want to keep it like this, I'd be open to talking about it, otherwise I would recommend making the switch